src/Raven.Studio/typescript/viewmodels/resources/resources.ts

```diff
@@ -311,6 +314,15 @@ class resources extends viewModelBase {
 
         if (matchedResource) {
             matchedResource.disabled(result.disabled);
+
+            // If Enabling a resource (that is selected from the top) than we want it to be Online(Loaded)
+            if (matchedResource.isCurrentlyActiveResource() && !matchedResource.disabled()) {
```

Comment:
don't construct database instance, call asResource instead. It will return database.